exchanges/request/request_test.go

```diff
@@ -343,13 +340,9 @@ func TestDoRequest_Retries(t *testing.T) {
 	require.NoError(t, err, "New requester must not error")
 
 	const numReqs = 4
```

Comment:
Since neqRegs is now only used in the for loop, can move it to there

Reply:
https://github.com/thrasher-corp/gocryptotrader/pull/2090/commits/4f08d17468a228f0911a4bcada4e7efd77187175